### ISSUE TITLE
Ajout « Fonction inverse » dans le plugin relais filaire

### ISSUE THREAD

**Issue by moebius911** [ldleman/yana-server#289]
Salut

J’aimerai savoir s’il est possible d’apporter une option supplémentaire pour le plugin wireRelay.
J’ai récemment fais l’acquisition d’une carte relais, mais malheureusement elle fonctionne en logique inverse.
Est-ce qu’il serait possible d’ajouter une option lors de la création des relais filaire pour indiquer si l’état repos est 1 ou 0 ?

Merci d’avance

**Comment by victor81000** [comment]
effectivement, je crois que nous sommes legion a acheter ces modules pas cher pour essayer, pour l'instant je me contente d'inverser les commandes mais c'est vrai que cette option serait interressante

**Comment by ldleman** [comment]
Normalement il y a un chevalet sur le relais en luis même permettant de modifier l'état de repos c'est pourquoi ce n'est pas prévu coté code.

Le chevalet est très souvent discret, il chevauche deux pin sur une rangée de trois ou 4 , il est nécessaire de le décaler d'un pin pour changer l'état initial.

**Comment by victor81000** [comment]
sur le mien, je n'ai pas ces chevalets, mais c'est un relais filaire sodial, les bleus

**Comment by ldleman** [comment]
C'est ajouté, à tester :)

**Comment by victor81000** [comment]
yeeeees!! je teste ca des que je rentre! :)

**Comment by victor81000** [comment]
je viens d'essayer, mais yana refuse de cooperer ( une phrase du style qu'elle ne me connait pas, est peinée et qu'elle refuse de faire ça, je ne lui pourtant pas fait de proposition indecente XD )
ca marchait avant la modification et si je passe par commande vocale ( en mettant a 0 pour allumer), cela fonctionne.
a la limite, plutot que de modifier le code, mettre juste un avertissement comme quoi pour les relais filaires sodial ou autres clones, qui ont une logique inversée, il faut mettre la phrase pour rendre actif sur la commande "off" et inversement, cela ne fait certes pas tres propre, mais c'est clair pour le neophyte XD.

**Comment by victor81000** [comment]
ca maaaaarche!!! :) merci pour la modification :)

**Comment by ldleman** [comment]
Curieux, je n'ai rien fait de plus depuis 9 jours ^^, tu as changé quelque chose ?

**Comment by victor81000** [comment]
rien depuis mon dernier message? Oo'
j'ai reinstallé sans mettre les mises a jour jusqu'a hier
les modif sont juste la mise en place de motion et motioneye  (les plugins stream et photo pi ne fonctionne pas chez moi  ...XD....quoi le boulet non mais ho! XD
mais maintenant elle coopere et ne s'excuse plus....vas savoir, les mystere de l'informatique.....

**Comment by ldleman** [comment]
Ben non ^^, peut être que tu avais un pb de cache ou de mise à jour mal passée ?
Ou alors yana était juste grognonne lors de ton premier test, parfois je la complique tellement que je ne la comprends plus :D.
